README.md

```diff
@@ -1941,6 +1941,51 @@ Usage typically looks like:
     }
 ~~~
 
+##### `kerb`
```

Comment:
Could you rename this to `auth_kerb` to match the name of the module? It seems like that would be more consistent.